src/fsharp/env.fs

```diff
@@ -405,13 +405,14 @@ type public TcGlobals =
       bitwise_or_vref            : ValRef;
       bitwise_and_vref           : ValRef;
       bitwise_xor_vref           : ValRef;
-      bitwise_unary_not_vref     : ValRef;
+      bitwise_unary_logical_not_vref     : ValRef;
```

Comment:
Is the naming of this correct? I always thought bitwise and logical were mutually exclusive.